admin_manual/configuration_user/two_factor-auth.rst

```diff
@@ -76,3 +83,14 @@ The associations of removed providers can be cleaned up via :ref:`occ <occ>`::
  sudo -u www-data php occ twofactorauth:cleanup <provider_id>
 
 .. warning:: This operation is irreversible. Only run it for providers you do not intend to enable again.
+
+Disabling two-factor authentication
```

Comment:
For me the use case is, that a user has forgotten/lost their second factor. If there are different factors possible, I'd add a line to show 2FA status of the user first:
`sudo -u www-data php occ twofactorauth:state <uid>`